src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationModal.js

```diff
@@ -52,19 +53,35 @@ const ImageAnnotationModal = ({
   const [hasMapLoaded, setHasMapLoaded] = useState(false)
   const [isTableShowing, setIsTableShowing] = useState(true)
   const map = useRef(null)
+  const [patchesGeoJson, setPatchesGeoJson] = useState()
 
```

Comment:
There is some existing inconsistency with the name for points/patches. Im going to start using 'patches' which exists in some places for the boxes we see in the image classification image. Its less generic and more 'sticky' than points I think....